deploy/docker/tests/test-pg-auth.sh

```
@@ -0,0 +1,186 @@
+        echo "Appsmith instance failed to start."
+        exit 1
+    fi
+
+    # Update the APPSMITH_DB_URL in docker.env to point to postgres to initialise appsmith user and schema when the container with local image is started
+    update_db_url
+    echo "Remove container to reuse the same volume for local image"
+    docker compose down --timeout 30 # wait upto timeout for graceful shutdown.
+    # ensure the container exists before trying to remove it
+    docker compose ps -q "${container_name}" && \
+        docker compose rm -fsv "${container_name}" || \
+        echo "Container "${container_name}" does not exist."
```

Shellcheck

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)